### DISCUSSION TITLE
Github Student Development Pack Application Rejected

### DISCUSSION THREAD

**Discussion by ghost** [community/community#23132] [New to GitHub]
<p>I’m told that</p>
<blockquote>
<p>The item you uploaded is insufficient to demonstrate your current academic status. Modifications to your photo proof might have been detected by the system. Please see suggestions.</p>
</blockquote>
<p>I am uploading my current student ID card with a current enrollment date as a document for proof. It is included as follows:</p>
<p>[IMAGE REMOVED DUE TO PERSONAL INFORMATION]</p>
<p>I am applying with my school email as well, where the school name matches that on my student ID card:</p>
<p>[IMAGE REMOVED DUE TO PERSONAL INFORMATION]</p>
<p>However, I was rejected for 3 times without explanation :frowning:</p>
<p>May the staff help me check my pending, please? I will really appreciate that :smiley:</p>

**Comment by queenofcorgis** [comment, marked as answer]
<p>For student benefits and related conversations, we recommend visiting the <a href="https://github.com/orgs/github-community/discussions/9">Student Developer Pack FAQs post</a> as that may help answer your questions. If not, please use that Discussion to ask for assistance as that is the Education Community’s new home.</p>
